api/bin/create_erds.py

```diff
@@ -19,7 +20,9 @@
 ERD_FOLDER = pathlib.Path(__file__).parent.resolve()
 
 # If we want to generate separate files for more specific groups, we can set that up here
-ALL_MODULES = [opportunity_models]
+ALL_MODULES = [opportunity_models, transfer_topportunity_models]
+
+TRANSFER_TABLE_MODULES = [transfer_topportunity_models]
```

Comment:
(Tiny) Could use tuples as they are constants

```suggestion
ALL_MODULES = (opportunity_models, transfer_topportunity_models)

TRANSFER_TABLE_MODULES = (transfer_topportunity_models,)
```